models/user.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Table, Column
+from config.db import meta, engine
+from sqlalchemy.sql.sqltypes import INTEGER, VARCHAR, SMALLINT, TEXT, DateTime
+from typing import Optional
+from pydantic import BaseModel
+
+userTable = Table("users", meta,
+                  Column("id", INTEGER, primary_key=True),
+                  Column("username", VARCHAR(150)),
+                  Column("password", SMALLINT),
+                  Column("salt", VARCHAR(45)),
+                  Column("created_at", DateTime),
+                  Column("updated_at", DateTime),
+                  Column("deleted_at", DateTime)
+                )
+
+#meta.create_all(engine)
```

Comment:
is it still useful? if not, let's remove any unused commented line

Reply:
The command was used when the table does not exists in DB, but is unuseful when the table is created trought the migration.